app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ]   You should have show method  in post controller, as it is a part of the requirement as well. The show method help us to see the specific post as the index list all the post in general. so it is very important to implement it as well in your project.